src/components/AddBookingForm/AddBookingForm.jsx

```diff
@@ -23,21 +38,46 @@ const AddBookingForm = ({ bookings, setBookings }) => {
     setFormData((prevData) => ({ ...prevData, [name]: value }));
   };
 
+  const validateBooking = () => {
+    if (!isNaN(parseInt(formData.firstName))) {
+      alert("Enter a valid name");
+    } else if (!isNaN(parseInt(formData.surname))) {
+      alert("Enter a valid surname");
+    } else if (
+      !(
+        formData.email.includes("@") &&
+        formData.email.includes(".") &&
+        formData.email.indexOf("@") === formData.email.lastIndexOf("@") &&
+        formData.email.indexOf("@") < formData.email.lastIndexOf(".")
+      )
+    ) {
+      alert("Enter a valid email");
+    } else if (!(formData.roomId > 0 && formData.roomId < 100)) {
+      alert("Enter a valid roon id");
+    } else if (!(formData.checkInDate < formData.checkOutDate)) {
+      alert("Enter valid dates");
```

Comment:
Good work, a more specfic error message here would be helpful to the user though - doesn't say which date is invalid